### ISSUE TITLE
OldStyleASAPMiddleware.__init__() is broken

### ISSUE THREAD

**Issue by seanconaty** [atlassian/asap-authentication-python#101]
We're using this by way of the `ProxiedAsapMiddleware`

```
    def __init__(self):
        self.backend = DjangoBackend()
        self.settings = self.backend.settings
>       self._verifier = backend.get_verifier(settings=self.settings)
E       NameError: global name 'backend' is not defined

/usr/lib/python2.7/site-packages/atlassian_jwt_auth/frameworks/django/middleware.py:29: NameError
```

**Comment by dbaxa** [comment]
@seanconaty hmm, it looks like we don't have test coverage for `OldStyleASAPMiddleware`.

**Comment by dbaxa** [comment]
This should be fixed by #102 .
